### ISSUE TITLE
Feature request - Simultaneous refresh of feeds

### ISSUE THREAD

**Issue by Xmasjos** [poetaster/tidings#64]
GPodder recently added a [feature](https://github.com/gpodder/gpodder-sailfish/pull/150) for simultaneous refresh of podcast feeds.
It would be quite nice to have a similar feature in tidings, but I have no idea about how to implement it (haven't done any QML / C++ development since my Symbian days).

**Comment by poetaster** [comment]
I'll look at this

from concurrent.futures import ThreadPoolExecutor

**Comment by poetaster** [comment]
Ok, it looks like there is already primitive async setup in place with

BackgroundWorker.qml (used by NewsBlendModel.qml). It seems to be queuing url fetches with a slight (30ms) offset. 

But there are simultaneous actions like removing read items, updating stats, AND adding ListItems that happen while loading. So it looks like it is happening in parallel but not efficiently. Looks like it needs rebuilding of the 
NewsBlendModel to, possibly, defer some things. 

But I'm not certain yet. I also need to add debug flags since the app is constantly spewing debug at the console.

**Comment by poetaster** [comment]
Nope. I'm wrong. It's serial. On it.
